### DISCUSSION TITLE
Clickable debug errors that take you to a Godot Q&A page for that general error

### DISCUSSION THREAD

**Discussion by jobieadobe** [godot-extended-libraries/godot-ideas#50] [Ideas]
So many times I don't even understand what the error message is even meaning! But how nice would it be (especially for novice programmers) if we could click that error and it take you to a default Godot Q&A page (or something like that) where it explains what that general error means and you can see posted solutions from tons of other people who have already had the error! I feel like this would make Godot way more accessible for new programmers as well as cut down on duplicate questions being asked over and over and over!

**Comment by Xrayez** [comment]
See related proposal:

- godotengine/godot-proposals#481
